collect_app/src/main/java/org/odk/collect/android/widgets/BarcodeWidget.java

```diff
@@ -147,7 +140,7 @@ public void denied() {
     private void setCameraIdIfNeeded(IntentIntegrator intent) {
         String appearance = getFormEntryPrompt().getAppearanceHint();
         if (appearance != null && appearance.equalsIgnoreCase(WidgetAppearanceUtils.FRONT)) {
```

Comment:
Would be good to move this code to `WidgetAppearanceUtils` and have `isFrontCamera()` method there for example which you could then use here.